### ISSUE TITLE
PureComponent support

### ISSUE THREAD

**Issue by alexgvozden** [xabikos/vscode-react#31]
Do you plan to add PureComponent support?

Alex

**Comment by xabikos** [comment]
@alexgvozden sorry for the late reaction.

Are you by pure component referring to stateless component. In this case please have a look at `rsc` and `rscp` snippets. 

Otherwise could you provide some more info?

**Comment by alexgvozden** [comment]
sorry I uninstalled it but I mean not stateless component but PureComponent class 

I created my own 

```
"PureComponent": {
    "prefix": "pcomp",
    "body": [
      "import React, { PureComponent } from 'react';",
      "import PropTypes from 'prop-types';",
      "",
      "class $1 extends PureComponent {",
      "\tstatic propTypes = {};",
      "",
      "\tstatic defaultProps = {",
      "\t\t...PureComponent.defaultProps",
      "\t};",
      "",
      "\trender() {",
      "\t\treturn null;",
      "\t}",
      "}",
      "",
      "export default $1;"
    ],
    "description": "Basic PureComponent"
  }
```

**Comment by xabikos** [comment]
@alexgvozden I just published a new version [1.7.0](https://github.com/xabikos/vscode-react/releases/tag/1.7.0) that includes a similar snippet. Thanks for the idea!
